### PR TITLE
Add json schema to samples

### LINKED ISSUE #2008: Update element template documentation to feature JSON schema
__What should we do?__

Update documentation in various places to add a `$schema` property that hooks up the newly created [JSON schema](https://github.com/camunda/element-templates-json-schema):

* https://docs.camunda.org/manual/7.14/modeler/element-templates/
* https://github.com/camunda/camunda-modeler/tree/develop/docs/element-templates

__Why should we do it?__

The schema property allows editors to provide editor assistance.

### PR DESCRIPTION
**Pull request by MaxTru**
__Which issue does this PR address?__

Related to #2083 
Closes #2008

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: docs/element-templates/README.md [hunk 1]
```diff
@@ -110,8 +112,10 @@ Templates are defined in template descriptor files as a JSON array:
 
 As seen in the code snippet a template consist of a number of important components:
 
+* `$schema : String`: Optional URI pointing towards the [json schema](https://json-schema.org/) which defines the structure of the element template json. Element template schemas are maintained in the [element templates json schema](https://github.com/camunda/element-templates-json-schema) repository. Following the [json schema](https://json-schema.org/) standard, you may use them to for validation or to get assistance (e.g., auto-completion) when working with them in your favorite IDE.
```
Comment (pinussilvestrus) [inline docs/element-templates/README.md:115]: > you may use them to for validation

one "to" too much

> you may use them for validation

Comment (pinussilvestrus) [reply docs/element-templates/README.md:115]: If we use "json" as a word, I'd go with

> JSON
> JSON schema

to make it consistent with the rest of the docs.

**Review by pinussilvestrus — COMMENTED** [review]
Is this is also related to https://github.com/camunda/camunda-modeler/issues/2008?

File: docs/element-templates/README.md (hunk 1 above)
Comment (MaxTru) [reply docs/element-templates/README.md]: :+1: ty

**Comment by MaxTru** [comment]
> Is this is also related to #2008?

:+1: Good point.
I think it even closes it.

**Comment by MaxTru** [comment]
@pinussilvestrus or @philippfromme could you please have another look (or directly approve based on the conversation @pinussilvestrus and I had in this issue :-) )

**Review by pinussilvestrus — APPROVED** [review]

**Pull request merged**